FAQ/index.html

```diff
@@ -11,6 +11,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta charset="UTF-8" />
 
+    <meta property="og:site_name" content="NewPipe">
+    <meta property="og:title" content="NewPipe - a free YouTube client">
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{ site.url }}">
+    <meta property="og:image" content="{{ site.baseurl/img/logo_400.png }}">
```

Comment:
```HTML
<meta property="og:image" content="{{ site.url }}/img/logo_400.png">
```

Reply:
:man_facepalming: so stupid of me :sweat_smile:

Reply:
wait, should it be `url` or `baseurl`?
wouldn't baseurl be `https://newpipe.schabi.org/` and url `https://newpipe.schabi.org/blog/`?

Reply:
Oh my bad.

Reply:
It took me some time to get the OG preview. It showed me, that I wasn't completely wrong and you either :laughing: 
Absolute links are required to show thumbnail. For this reason, this version is the correct one:
``` HTML
<meta property="og:image" content="{{site.url}}{{ site.baseurl }}/img/logo_400.png">
```

Reply:
not entirely sure what you said, but I updated it to your version.
I use "Open Graph Preview" as an chromium extension to preview og

Reply:
Yes, I use it too. Did it work before these changes? It didn't work for me.